packages/bentley/grouping-mapping-widget/README.md

```diff
@@ -0,0 +1,25 @@
+# @bentley/grouping-mapping-widget
+
+Copyright © Bentley Systems, Incorporated. All rights reserved.
+
+The Grouping Mapping widget is a UI component for iTwin Viewer applications that simplifies how users (and developers) interface with the [Reporting Platform APIs](https://developer.bentley.com/apis/insights/overview/).
```

Comment:
are there any scopes required to use this?

Reply:
good call, yeah can add that documentation here